sdk/documenttranslation/azure-ai-documenttranslation/samples/async_samples/sample_batch_translation_async.py

```diff
@@ -94,13 +94,13 @@ async def check_documents(self, client, job_id):
         async for document in doc_statuses:
             if document.status == "Failed":
                 print("Document at {} failed to be translated to {} language".format(
-                    document.url, document.translate_to
+                    document.translated_document_url, document.translate_to
                 ))
                 print("Document ID: {}, Error Code: {}, Message: {}".format(
                     document.id, document.error.code, document.error.message
                 ))
-                if document.url not in docs_to_retry:
-                    docs_to_retry.append(document.url)
+                if document.translated_document_url not in docs_to_retry:
```

Comment:
I think we want to save the `source_url` for the document that failed (so we can retry on it), the `translated_document_url` shouldn't exist in this scenario.

Reply:
yeah, i mentioned it here
https://github.com/Azure/azure-sdk-for-python/issues/17288#issuecomment-804441848